config/rubocop.yml

```diff
@@ -61,3 +61,20 @@ ExtraSpacing:
 # Buggy
 FormatParameterMismatch:
   Enabled: false
+
+# Align if/else blocks with the variable assignment
+EndAlignment:
+  AlignWith: variable
+
+# Use square brackets for literal Array objects
+PercentLiteralDelimiters:
+  PreferredDelimiters:
+    '%':  '{}'
+    '%i': '[]'
+    '%q': ()
+    '%Q': ()
+    '%r': '{}'
+    '%s': ()
+    '%w': '[]'
```

Comment:
Oh nice. I like that you turned my preference noted in another comment into a Rubocop setting.